ddl/000024_actions_auth_policies.up.sql

```diff
@@ -0,0 +1,11 @@
+BEGIN;
+
+UPDATE auth_policies
+SET statement = statement || '{"Action": ["ci:Read*"], "Effect": "allow", "Resource": "*"}'::jsonb
+WHERE display_name = 'RepoManagementReadAll' AND NOT statement @> '[{"Action": ["ci:Read*"], "Effect": "allow", "Resource": "*"}]'::jsonb;
```

Comment:
Shouldn't you switch between the non-array and array values?
```suggestion
UPDATE auth_policies
SET statement = statement || '[{"Action": ["ci:Read*"], "Effect": "allow", "Resource": "*"}]'::jsonb
WHERE display_name = 'RepoManagementReadAll' AND NOT statement @> '{"Action": ["ci:Read*"], "Effect": "allow", "Resource": "*"}'::jsonb;
```

Reply:
both work but your is better :)